src/test/java/hudson/plugins/sonar/SonarTestCase.java

```diff
@@ -76,8 +76,12 @@
   protected Maven.MavenInstallation configureDefaultMaven() throws Exception {
     File mvn = new File(getClass().getResource("SonarTestCase/maven/bin/mvn").toURI().getPath());
     if (!Functions.isWindows()) {
-      // noinspection OctalInteger
-      GNUCLibrary.LIBC.chmod(mvn.getPath(), 0755);
+      try {
+        // noinspection OctalInteger
+        GNUCLibrary.LIBC.chmod(mvn.getPath(), 0755);
```

Comment:
see javadoc from GNUCLibrary:
> Not available on all platforms so the caller should recover gracefully in case of {@link LinkageError}

this currently fails on Mac M1, adding a try catch appears to be enough for tests to pass here fine